src/index.html

```diff
@@ -5,8 +5,468 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="index.html">
+        <img src="./images/logo.png" alt="Moyo-logo">
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
```
don't forget to add spaces between one-level elements